app/src/main/java/nl/tudelft/b_b_w/model/Block.java

```diff
@@ -18,17 +18,16 @@
 
     /**
      * @param _owner              owner of a block
-     * @param _sequenceNumber     sequence number of the block
      * @param _ownHash            our own hash
      * @param _previousHashChain  the hash value of the block before in the chain
      * @param _previousHashSender the hash value of the block before of the sender
      * @param _publicKey          public key of the sender
      * @param _iban               IBAN number of a contact
      * @param _isRevoked          boolean to check whether a block is revoked or not
      */
-    public Block(String _owner, int _sequenceNumber, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban, boolean _isRevoked) {
+    public Block(String _owner, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban, boolean _isRevoked) {
         this.owner = _owner;
-        this.sequenceNumber = _sequenceNumber;
```

Comment:
Cant figure out a way to fix this, but I think as this pull request has a lot of changes, it should be merged so others can use this new develope branch while i can fix this independent issue